src/app/summary/summary.module.ts

```diff
@@ -0,0 +1,34 @@
+import {NgModule} from '@angular/core';
+import {CommonModule} from '@angular/common';
+import {ChartComponent} from "./chart/chart.component";
+import {ChartsModule} from "ng2-charts";
+import {MiniCardComponent} from "./mini-card/mini-card.component";
+import {SummaryComponent} from "./summary.component";
+import {SummaryService} from "./summary.service";
+import {FaIconLibrary, FontAwesomeModule} from "@fortawesome/angular-fontawesome";
+import {fas} from "@fortawesome/free-solid-svg-icons";
+
+@NgModule({
```

Comment:
I'm not sure this page warrants it's own module. I think it would be better placed inside the Projects module. It is using the ProjectsService and is a page about projects.

Reply:
It is declaring lots of components that are only used in the summary page. Why add all of them to the main module?